codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/AbstractDirectedCodegen.java

```diff
@@ -35,6 +37,31 @@
 
 @SmithyInternalApi
 public abstract class AbstractDirectedCodegen implements DirectedCodegen<GoCodegenContext, GoSettings, GoIntegration> {
+    public static void run(PluginContext context, DirectedCodegen<GoCodegenContext, GoSettings, GoIntegration> directedCodegen) {
```

Comment:
I was curious why we had this class just laying around. The only implementation of it was the `ServerCodegenPlugin` to generate the server code of a model, which is still a work in progress from what I know

Reply:
Correct, server codegen was bootstrapped as a hackathon project in I think 2024.